specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/vmssPublicIpAddress.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "NetworkManagementClient",
     "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
-    "version": "2017-03-30"
+    "version": "2018-06-01"
```

Comment:
Should it be `2019-06-01`?

Reply:
When I tried 2019-06-01, I get this error - 

**Get-AzureRmResource : NoRegisteredProviderFound : No registered resource provider found for location 'westus' and API version '2019-06-01' for type 'virtualMachineScaleSets/publicIPAddresses'. The supported api-versions are '2017-03-30, 2017-12-01, 2018-04-01, 2018-06-01, 2018-10-01'. The supported locations are 'eastus, eastus2, westus, centralus, northcentralus, southcentralus, northeurope, westeurope, eastasia, southeastasia, japaneast, japanwest, australiaeast, australiasoutheast, australiacentral, brazilsouth, southindia, centralindia, westindia, canadacentral, canadaeast, westus2, westcentralus, uksouth, ukwest, koreacentral, koreasouth, francecentral, southafricanorth, uaenorth'.**

May be I can chage it to '2018-10-01'

Reply:
It's your choise which api version you need

Reply:
I think in this case you should use the latest one